hashicorp/dev-portal
# chore: removes link warning

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-BRANCH_NAME-hashicorp.vercel.app/PATH_TO_VIEW) 🔎
- [Asana task](url) 🎟️

## 🗒️ What

In local dev, many toasts were popping to show this warning. I believe since this component supports both options we shouldn't show this warning toast as it degrades developer experience. Using the target attribute should be an open preference and the cognitive load could go both ways if one isn't familiar with the prop structure of this link component. 

## To Test

Pull things down locally, start the server and you should see no link warnings pop.
